website/sidebars.js

```diff
@@ -235,6 +236,7 @@ module.exports = {
 						'rules',
 						'runpathdirs',
 						'runtime',
+						'sanitize',
```

Comment:
This fix is unrelated to this PR ;-)

Reply:
Yeah, I just had it locally and didn't catch it.  Want me to back this out and put in a separate PR?

Reply:
As you prefer.

but if you keep it, you have to uncheck:
- [ ] Focus on a single fix or feature; remove any unrelated formatting or code changes
 ;-)

Reply:
Unchecked